pkg/engine/engine.go

```diff
@@ -115,6 +115,8 @@ func (e *Engine) Render(chrt *chart.Chart, values chartutil.Values) (map[string]
 type renderable struct {
 	// tpl is the current template.
 	tpl string
+	// path is the path of chart
+	path string
```

Comment:
Yes, but my understand is that if you have this path, then you can extract it? e.g. 
`charts/chartX/charts/ChartY/Charts/ChartZ/templates/xxx.yaml` --> `chartX/chartY/chartZ`

Not a big deal though, will merge if we agree on this.